src/main/java/no/nav/veilarbvedtaksstotte/client/arena/VeilarbarenaClientImpl.java

```diff
@@ -39,8 +41,9 @@ public VeilarbarenaClientImpl(String veilarbarenaUrl, Supplier<String> userToken
     @Cacheable(CacheConfig.ARENA_BRUKER_CACHE_NAME)
     public Optional<VeilarbArenaOppfolging> hentOppfolgingsbruker(Fnr fnr){
         Request request = new Request.Builder()
-                .url(joinPaths(veilarbarenaUrl, "/api/oppfolgingsbruker/", fnr.get()))
+                .url(joinPaths(veilarbarenaUrl, "/api/v2/oppfolgingsbruker/"))
```

Comment:
Tror ikke trailing-slash er tillatt sånn applikasjonene våres er satt opp. Uavhengig av dette så ville jeg fjernet det da det er en ganske vanlig konvensjon å ikke ha det.

Reply:
Gjelder forøvrig flere steder i koden.